gcp/rakefiles/entrypoint.rake

```diff
@@ -131,23 +131,23 @@ end
 
 desc '[ADVANCED] Destroy secrets file stored in GS bucket for encryption key, passed as argument -- rake destroy_secrets"[default]"'
 task :destroy_secrets, [:encryption_key] => [:set_vars] do |taskname, args|
-  if args[:encryption_key].nil?
```

Comment:
Looks like we require `encryption_key` below, so why is this check removed?

Also I don't see a default value for `encryption_key`, so it doesn't look like this will work with the `destroy_secrets` command in gitlab-ci.yml?

Reply:
Without `encryption_key` it will delete secrets in all buckets, with `encryption_key` – only in a bucket for that particular key. It works as expected without arguments.